Mesh_3/examples/Mesh_3/mesh_3D_gray_vtk_image.cpp

```diff
@@ -0,0 +1,79 @@
+
+#include <vtkImageData.h>
+#include <vtkDICOMImageReader.h>
+#include <vtkImageReader.h>
+#include <vtkImageGaussianSmooth.h>
+#include <vtkDemandDrivenPipeline.h>
+
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Mesh_triangulation_3.h>
+#include <CGAL/Mesh_complex_3_in_triangulation_3.h>
+#include <CGAL/Mesh_criteria_3.h>
+
+#include <CGAL/Gray_image_mesh_domain_3.h>
+#include <CGAL/make_mesh_3.h>
+#include <CGAL/Image_3.h>
+#include <CGAL/read_vtk_image_data.h>
+
+typedef short Image_word_type;
+
+// Domain
+typedef CGAL::Exact_predicates_inexact_constructions_kernel K;
+typedef CGAL::Gray_image_mesh_domain_3<CGAL::Image_3,K, 
+                                       Image_word_type,
+                                       std::binder1st< std::less<Image_word_type> > > Mesh_domain;
+
+// Triangulation
+typedef CGAL::Mesh_triangulation_3<Mesh_domain>::type Tr;
+typedef CGAL::Mesh_complex_3_in_triangulation_3<Tr> C3t3;
+
+// Criteria
+typedef CGAL::Mesh_criteria_3<Tr> Mesh_criteria;
+
+// To avoid verbose function and named parameters call
+using namespace CGAL::parameters;
+
+int main(int argc, char* argv[])
+{
+  // Loads image
+   
+
+  vtkDICOMImageReader*dicom_reader = vtkDICOMImageReader::New();
+  dicom_reader->SetDirectoryName(argv[1]);
+  
```

Comment:
As-is this example will reddish the testsuite, because we do not have data test files.